packages/enzyme-test-suite/test/Debug-spec.jsx

```diff
@@ -626,6 +626,30 @@ describe('debug', () => {
   <span>
     span text
   </span>
+</div>`
+      ));
+    });
+
+    it('options.verbose causes boxed primitives to be unboxed', () => {
```

Comment:
None of these are boxed primitives; a boxed primitive is like `Object(2)`. Do you mean objects and arrays?

Reply:
Ah. tbh, i heard you say it once and I didn't really know what it meant, so like everyone, I just pretended it was the right thing. my bad. I'll fix up the wording to just mean objects and arrays.